third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -322,29 +311,31 @@ PjRtComputationClient::ExecuteComputation(
   execute_options.untuple_result = options.explode_tuple;
   execute_options.strict_shape_checking = false;
 
-  std::optional<PjRtFuture<Status>> returned_future;
+  // std::optional<PjRtFuture<Status>> returned_future;
   std::vector<std::unique_ptr<xla::PjRtBuffer>> results =
       pjrt_computation.executable
-          ->ExecuteSharded(buffers, pjrt_device, execute_options,
-                           returned_future, /*fill_future=*/true)
+          ->ExecuteSharded(buffers, pjrt_device, execute_options)
           .value();
+
   // Signal that `ExecuteSharded` has completed for the ExecuteTime metric.
   // Copies the `timed` shared pointer into the lambda.
-  returned_future->OnReady([timed](Status unused) mutable { timed.reset(); });
+  // TODO(wcromar): Uncomment this when we update past TF commit
+  // b8f59020ea0e9e6fba0e9c5e7be88271703eaf9e
+  // returned_future->OnReady([timed](Status unused) mutable { timed.reset();
```

Comment:
is `returned_future->OnReady` broken with current pin?

Reply:
Yes. The fix should be included in our next pin update and we can revert this change.